src/RuleSet/Sets/PERCS2x0Set.php

```diff
@@ -42,6 +42,10 @@ public function getRules(): array
             ],
             'method_argument_space' => true,
             'single_line_empty_body' => true,
+            'trailing_comma_in_multiline' => [
+                'after_heredoc' => true,
+                'elements' => ['arguments', 'arrays', 'match', 'parameters'],
+            ],
```

Comment:
> PER rule covers any kind of multiline list

I'm not sure. eg, it mentions function arguments in function declaration, but not function call params.

I would be careful with `@all` by default, as it can easily create bc breaks in someone CS.  I'm +1 to have `@all` available